<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="okBox">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>540</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1081</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Emettor power (dBm)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="typeComboBox">
      <item>
       <property name="text">
        <string>Concrete</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bricks</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Standard partition wall</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Wall thickness (cm)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Wall type</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="wallThickSpinBox">
      <property name="maximum">
       <number>500</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Amount of rebound (n)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QSpinBox" name="reboundSpin">
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QSpinBox" name="discretSubBox">
      <property name="maximum">
       <number>400</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Discretisation of the plane for level coverage (subdivision)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="powerSpinBox">
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="diffractBox">
      <property name="text">
       <string>Taking diffraction into account (integral approximation model, very calculation heavy)</string>
      </property>
      <property name="tristate">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="resetDefaultsButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>540</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Defaults</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyChangesButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>540</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply changes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okBox</sender>
   <signal>accepted()</signal>
   <receiver>settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okBox</sender>
   <signal>rejected()</signal>
   <receiver>settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
